### ISSUE TITLE
Weird error with React and Browserify

### ISSUE THREAD

**Issue by adityan-webonise** [airbrake/airbrake-js#259]
Getting an error:

Error while running `/mypath/node_modules/.bin/browserifyinc -d -t babelify --extension=".jsx" --list --cachefile=/mypath/tmp/cache/browserify-rails/browserifyinc-cache.json -o "/mypath/tmp/cache/browserify-rails/output20170217-12647-1ys2uqp" -`:

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'stackframe' from '/mypath/node_modules/airbrake-js/dist'
    at /mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /mypath/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:111:15)

**Comment by vmihailenco** [comment]
Which version are you using?

**Comment by vmihailenco** [comment]
Please try again with recently released v0.6.0

**Comment by davebrace** [comment]
I'm running into this error as well when trying to use `0.6.0` or `0.8.2`. `0.5.9` does not error.

```
Cannot find module 'stackframe' from '/Users/davebrace/projects/charlie-xray/node_modules/airbrake-js/dist'
```

**Comment by vmihailenco** [comment]
I need some way to reproduce this. How do you require airbrake-js? We have browserify example that works just fine https://github.com/airbrake/airbrake-js/blob/master/examples/browserify/app.js

**Comment by davebrace** [comment]
We're using it similar to that example:

```js
const AirbrakeClient = require('airbrake-js');
const AirbrakejQueryInstrumentation = require('airbrake-js/lib/instrumentation/jquery');
```

For what its worth, I'm using node v7.7.3. Not sure if that could be the issue as I noticed your travis config is set to test with node v6.1.

**Comment by vmihailenco** [comment]
Try to upgrade to v0.8.2 and remove `const AirbrakejQueryInstrumentation = require('airbrake-js/lib/instrumentation/jquery');`. jQuery instrumentation is gone.

**Comment by davebrace** [comment]
I'll give that a try. What is the recommendation for handling errors in jQuery callbacks now that the jQuery instrumentation is gone?

**Comment by davebrace** [comment]
I tried using v0.8.2 and removing the jQuery instrumentation, but I'm still get the same error:

```
Error: Cannot find module 'stackframe' from '/project_path/node_modules/airbrake-js/dist'
```

**Comment by adityan-webonise** [comment]
Sorry for being late. The issue still persists for me in version 0.8.3

**Comment by adityan-webonise** [comment]
Regarding [reproducing the error](https://github.com/airbrake/airbrake-js/issues/259#issuecomment-287816577), in your example, it seems like you are using `grunt-browserify` [here](https://github.com/airbrake/airbrake-js/blob/master/examples/browserify/package.json#L6), probably can be reproduced using just `browserify`?

**Comment by adityan-webonise** [comment]
Also, solution might be similar to https://github.com/commissure/redbox-react/issues/17

**Comment by paulgabronis** [comment]
Same here:
`>> Error: Cannot find module 'stackframe' from '/scripts/libs', `
if I require airbrake from npm modules got the same problem.
Any real solution?

**Comment by vmihailenco** [comment]
How do you require it?

**Comment by paulgabronis** [comment]
@vmihailenco just like this:
`var AirbrakeJs = require('airbrake-js');`

**Comment by vmihailenco** [comment]
Hopefully fixed in v0.8.7
